powerstripflocker.tac

```diff
@@ -20,5 +21,10 @@ def getAdapter():
 
 application = service.Application("Powerstrip Flocker Adapter")
 
-adapterServer = internet.UNIXServer("/usr/share/docker/plugins/flocker.sock", getAdapter())
+DOCKER_PLUGINS_FOLDER = os.environ.get("DOCKER_PLUGINS_FOLDER")
```

Comment:
spell this `os.environ.get("DOCKER_PLUGINS_FOLDER", "/usr/share/docker/plugins") + "/flocker.sock"` to (a) fix a logic bug and (b) shorten the code